src/downshift.js

```diff
@@ -688,15 +685,13 @@ class Downshift extends Component {
     return `${this.props.id}-item-${index}`
   }
 
-  getItemProps = (
-    {
-      onMouseEnter,
-      onClick,
-      index,
-      item = requiredProp('getItemProps', 'item'),
-      ...rest
-    } = {},
-  ) => {
+  getItemProps = ({
+    onMouseEnter,
+    onClick,
+    index,
+    item = requiredProp('getItemProps', 'item'),
+    ...rest
+  } = {}) => {
```

Comment:
I don't know why this changed. It got a different prettier version somehow or something?